### PR TITLE
Sorting

### PR DESCRIPTION
**Pull request by pauladkisson**
Adds custom sorting

See example file: https://drive.google.com/file/d/1P7RgPE3aT9zVMbWaeEiVv7i7N6uBKokZ/view?usp=drive_link

### CODE REVIEW

File: src/jadhav_lab_to_nwb/olson_2024/olson_2024_sorting_interface.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    def __init__(self, spike_times_folder_path: DirectoryPath, unit_stats_folder_path: DirectoryPath):
+        super().__init__(spike_times_folder_path=spike_times_folder_path, unit_stats_folder_path=unit_stats_folder_path)
+
+    def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict):
+        spike_times_folder_path = Path(self.source_data["spike_times_folder_path"])
+        unit_stats_folder_path = Path(self.source_data["unit_stats_folder_path"])
+        nwbfile.add_unit_column(name="nTrode", description="The tetrode number for this unit")
+        nwbfile.add_unit_column(name="unitInd", description="The integer id each unit within each nTrode")
+        nwbfile.add_unit_column(
+            name="global_id", description="The global id for each unit: nTrode{nTrode}_unit{unitInd}"
```
Comment (alessandratrapani) [inline src/jadhav_lab_to_nwb/olson_2024/olson_2024_sorting_interface.py:24]:
```suggestion
            name="global_id", description=f"The global id for each unit: nTrode{nTrode}_unit{unitInd}"
```

File: src/jadhav_lab_to_nwb/olson_2024/olson_2024_sorting_interface.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        )
+        nwbfile.add_unit_column(name="nWaveforms", description="Number of waveforms (spikes) for each unit.")
+        nwbfile.add_unit_column(
+            name="waveformFWHM", description="Full width at half maximum of the template waveform in ms."
+        )
+        nwbfile.add_unit_column(
+            name="waveformPeakMinusTrough", description="Peak minus trough of the template waveform in uV."
+        )
+
```
Comment (alessandratrapani) [inline src/jadhav_lab_to_nwb/olson_2024/olson_2024_sorting_interface.py:33]: Minor comment: I noticed column names are written using camel case, but `global_id` in snake case. Is that intentional?

**Review by alessandratrapani — APPROVED** [review]
looks good only minor comments

File: src/jadhav_lab_to_nwb/olson_2024/olson_2024_sorting_interface.py (hunk 1 above)
Comment (pauladkisson) [reply src/jadhav_lab_to_nwb/olson_2024/olson_2024_sorting_interface.py]: thx for the catch

**Pull request merged**